### ISSUE TITLE
What is the memory requirement of Transformer TTS model?

### ISSUE THREAD

**Issue by erogol** [espnet/espnet#1264]
In [Mozilla TTS](https://github.com/mozilla/TTS) I try to keep model sizes reasonable and therefore my emphasis is mostly Tacotron. I see here people obtain good results with transformers but how does it compare with Tacotron in terms of training requirements?

My guess is that since spec sequences are too long, especially the decoder part should take a lot of memory at training time which would reduce the batch size.

**Comment by kan-bayashi** [comment]
Hi @erogol.
In terms of training, Transformer requires large batch size (>64 according to original paper), requiring large GPU memory.
And as you said, sometimes there is a very long sentence which causes OOM easily.
To address these issues, we support dynamic batch making and gradient accumulation.
(See https://github.com/espnet/espnet#how-to-set-minibatch about batch making)
By using these techniques, we can stably train Transformer with only a single GPU.
